packages/cli/src/validate.ts

```diff
@@ -28,41 +28,12 @@ export const isFieldValid = {
   },
   exports(pkg: Package): boolean {
     const generated = validFields.exports(pkg);
-    if (!equal(pkg.json.exports, generated)) {
-      return false;
-    }
-    // make sure conditions are in proper order
-    if (pkg.json.exports) {
-      const packageNames = Object.keys(generated);
-      return packageNames.every((pkgName) => {
-        const generatedConditions = generated[pkgName];
-        if (typeof generatedConditions !== "object") {
-          return true;
-        }
-        return assertKeysAreInSameOrder(
-          generatedConditions,
-          pkg.json.exports[pkgName]
-        );
-      });
-    }
-    return true;
+
+    // JSON.stringify to make sure conditions are in proper order
+    return JSON.stringify(pkg.json.exports) === JSON.stringify(generated);
```

Comment:
q: is a specific error raised for a mismatch anywhere?

Reply:
Not sure I understand totally the question? We error if the written exports field is different to the one that we generate in values or ordering